source/simulator/parameters.cc

```diff
@@ -1265,6 +1287,18 @@ namespace aspect
         AssertThrow (false, ExcNotImplemented());
     }
 
+    {
```

Comment:
Can you put this into a separate subsection ("Temperature field"?), and then move it to above or below where the same is done for composition?